lib/common/common_helper.rb

```diff
@@ -78,13 +80,39 @@ def add_trailing_slash(url)
   url =~ /\/$/ ? url : "#{url}/"
 end
 
-def missing_db_file?
+def missing_db_files?
   DbUpdater::FILES.each do |db_file|
     return true unless File.exist?(File.join(DATA_DIR, db_file))
   end
   false
 end
 
+# Find data.zip?
+def has_db_zip?
+  return File.exist?(DATA_FILE)? true : false
+end
+
+# Extract data.zip
+def extract_db_zip
+  Zip::File.open(DATA_FILE) do |zip_file|
+    zip_file.each do |f|
+      # Feedback to the user
+      #puts "[+] Extracting: #{File.basename(f.name)}"
+      f_path = File.join(DATA_DIR, File.basename(f.name))
+
+      # Create folder
+      FileUtils.mkdir_p(File.dirname(f_path))
```

Comment:
can we move this up to the top and just create the `DATA_DIR`? There are currently no plans in having subfolders in the data directory

Reply:
Fair enough.
Was trying to future proof the code, but guess it makes it more complex.

Altered